doc/user/layouts/shortcodes/views-indexes/index-query-optimization-specific-instances.html

```diff
@@ -0,0 +1,9 @@
+Specific instances where indexes can be useful to improve performance include:
+
+- When used in ad-hoc queries.
+
+- When used by multiple queries within the same cluster.
+
+- When used to enable delta joins.
+
+For more information, see [Optimization](/transform-data/optimization).
```

Comment:
It'd be nice to explain a bit more how indexes help in each of these situations! For example, the average user will not know what a delta join is, or why they should care.

Reply:
💯 Agreed.  Will actually do that when we update the transform-data/optimization page.